src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/services/CamundaService.java

```diff
@@ -239,20 +239,30 @@ public void completeTask(String taskId,
             eventVariable.put("eventId", new CamundaVariable(searchEventAndCase.getEventId(), "String"));
             Map<String, Map<String, CamundaVariable>> dmnRequest = new HashMap<>();
             dmnRequest.put("variables", eventVariable);
-            // A List (Array) with a map (One object) with objects inside the object (String and CamundaVariable).
-            List<Map<String, CamundaVariable>> evaluateDmnResult =
-                camundaServiceApi.evaluateDMN(
-                    authTokenGenerator.generate(),
-                    getTableKey(searchEventAndCase.getCaseJurisdiction(), searchEventAndCase.getCaseType()),
-                    dmnRequest
-                );
+            List<Map<String, CamundaVariable>> evaluateDmnResult;
+            if(searchEventAndCase.getCaseJurisdiction().equals("ia") &&
+               searchEventAndCase.getCaseType().equals("asylum")) {
+                // A List (Array) with a map (One object) with objects inside the object (String and CamundaVariable).
+               evaluateDmnResult =
+                    camundaServiceApi.evaluateDMN(
+                        authTokenGenerator.generate(),
+                        getTableKey(searchEventAndCase.getCaseJurisdiction(), searchEventAndCase.getCaseType()),
+                        dmnRequest
+                    );
+            } else {
+                return Collections.emptyList();
```

Comment:
I’m not sure returning an empty list is the right response if the request is sending effectively invalid data.  I think for MVP we can say that we only accept the jurisdiction of ia and caseType or asylum .  So I think the response should be an http 400 Bad Request response with text explaining what is wrong.

The text can be: ```Please check your request.  This endpoint currently only supports the Immigration & Asylum service.```

Reply:
Should add a test for this scenario too.